modules/concertBidAdapter.md

```diff
@@ -0,0 +1,33 @@
+# Overview
+
+```
+Module Name: Concert Bid Adapter
+Module Type: Bidder Adapter
+Maintainer: support@concert.io
+```
+
+# Description
+
+Module that connects to Concert demand sources
+
+# Test Paramters
+```
+  var adUnits = [
+    {
+      code: 'desktop_leaderboard_variable',
+      mediaTypes: {
+        banner: {
+          sizes: [[1030, 590]]
+        }
+      }
+      bids: [
+        {
+          bidder: "concert",
+          params: {
+            partnerId: 'test_partner'
```

Comment:
Can this be a `partnerId` that will return data? When I used these params in a test page, I got this response:
```
{"bids":[],"debug":{}}
```
Also, all responses from your endpoint should be compressed (such as gzip, compress, deflate)